Subject: enum has eclipsed enum34
Author: Robbie Harwood (frozencemetery) <rharwood@club.cc.cmu.edu>
Forwarded: https://github.com/pythongssapi/python-gssapi/pull/84
--- a/setup.py
+++ b/setup.py
@@ -206,6 +206,13 @@
                           re.sub(r'\.\. code-block:: \w+', '::',
                                  open('README.txt').read())))
 
+install_requires = [
+    'decorator',
+    'six'
+]
+if sys.version_info < (3, 4):
+    install_requires.append('enum34')
+
 setup(
     name='gssapi',
     version='1.1.3',
@@ -259,9 +266,5 @@
         extension_file('password_add', 'gss_add_cred_with_password'),
     ]),
     keywords=['gssapi', 'security'],
-    install_requires=[
-        'enum34',
-        'decorator',
-        'six'
-    ]
+    install_requires=install_requires
 )
