Subject: Spelling error (compatability)
Author: Robbie Harwood (work) <rharwood@redhat.com>
Forwarded: https://github.com/pythongssapi/python-gssapi/pull/66
--- a/gssapi/raw/sec_contexts.pyx
+++ b/gssapi/raw/sec_contexts.pyx
@@ -647,7 +647,7 @@
 
     Returns:
         bytes: the output token (if remote deletion is requested).  Generally
-            this is None, but bytes for compatability.
+            this is None, but bytes for compatibility.
 
     Raises:
         MissingContextError
